### PR TITLE
Stop deleting group entries on switch connection

### PR DESCRIPTION
**Pull request by antoninbas**
There is no reason to treat groups any differently from flows, meters, etc.

Bump up version to v0.12.0.

### CODE REVIEW

File: ofctrl/ofSwitch.go [hunk 1]
```diff
@@ -158,9 +158,6 @@ func (self *OFSwitch) IsReady() bool {
 
 // Handle switch connected event
 func (self *OFSwitch) switchConnected() error {
-	if err := self.clearGroups(); err != nil {
-		return fmt.Errorf("fails to clear groups: %v", err)
-	}
```
Comment (tnqn) [inline ofctrl/ofSwitch.go:163]: Just to confirm, connecting/reconnecting to OVS will no longer make any change to OpenFlow entries, right?

**Review by tnqn — APPROVED** [review]
LGTM

File: ofctrl/ofSwitch.go (hunk 1 above)
Comment (antoninbas) [reply ofctrl/ofSwitch.go:163]: That's correct. AFAIK, this is the only place that modified OF entries directly. We inherited that from the original ofnet library from contiv, for reasons that are not clear, and we have kept it as a special case until now. For example, in Antrea, we rely on this behavior, while for meter entries we have custom logic to delete meter entries. I see no reason to treat them differently.

Someone else actually requested that change a while ago, and provided some extra context about contiv: https://github.com/antrea-io/ofnet/pull/9

Comment (tnqn) [reply ofctrl/ofSwitch.go:163]: Thanks for the explanation, makes sense to me.

**Pull request merged**